### PR TITLE
Enhance adoption email

### LINKED ISSUE #1068: Transfering ownership of a cookbook workflow needs love
Transferring ownership of a cookbook is a very tricky process at the moment.  If you put a cookbook up for adoption and someone tries to adopt it you get an e-mail announcing their intent (good).  That e-mail contains their e-mail, but not their username (not so good).  The e-mail contains no link to allow you to easily transfer ownership (also not so good).  If you go to Supermarket and transfer ownership you're presented with a search dialog that searches by username, but not the e-mail that was provided to you (really bad).  You kinda have to guess what their name might be based on their e-mail address.  It might work out for you, but it probably won't.

### PR DESCRIPTION
**Pull request by nellshamrell**
Fixes #1068

### CODE REVIEW

**Comment by nellshamrell** [comment]
Packaged this and verified that it works!

File: app/views/adoption_mailer/interest_email.html.erb [hunk 1]
```diff
@@ -1 +1,9 @@
-Your <%= @thing %>, <%= @name %>, which is currently up for adoption, got some interest today.  <%= mail_to @email %> is interested in adopting it. Please get in touch with them.
+Your <%= @thing %>, <%= @name %>, which is currently up for adoption, got some interest today.<br />
+<%= link_to(@name, "#{Rails.root}#{cookbook_path(@name)}") %>
+<br />
+<br />
+Here is the interested user's information:<br />
```
Comment (nathenharvey) [inline app/views/adoption_mailer/interest_email.html.erb]: Here is the information you need to know about the awesome Chef who is interested in adopting your <%= @thing %>.

File: app/views/adoption_mailer/interest_email.html.erb [hunk 2]
```diff
@@ -1 +1,9 @@
...
+Supermarket Username: <%= @adopting_username %><br />
+Email Address: <%= mail_to @email %><br />
+
+Please get in touch with them.
```
Comment (nathenharvey) [inline app/views/adoption_mailer/interest_email.html.erb]: Please get in touch with this Chef to discuss the potential adoption.

**Comment by nellshamrell** [comment]
Great suggestions, @nathenharvey!  I've incorporated them in :)

File: app/views/adoption_mailer/interest_email.html.erb [hunk 3]
```diff
@@ -1 +1,9 @@
-Your <%= @thing %>, <%= @name %>, which is currently up for adoption, got some interest today.  <%= mail_to @email %> is interested in adopting it. Please get in touch with them.
+Your <%= @thing %>, <%= @name %>, which is currently up for adoption, got some interest today.<br />
+<%= link_to(@name, "#{Rails.root}#{cookbook_path(@name)}") %>
+<br />
+<br />
+Here is the information you need to know about the awesome Chef who interested in adoption your <%= @thing %>:<br />
```
Comment (nathenharvey) [inline app/views/adoption_mailer/interest_email.html.erb]: 'in adoption your'  should be 'in adopting your'

or "in adoptin' your" ;)

Comment (nellshamrell) [inline app/views/adoption_mailer/interest_email.html.erb]: oh, whoops!

Comment (nellshamrell) [inline app/views/adoption_mailer/interest_email.html.erb]: Fixed!

**Comment by nathenharvey** [comment]
:+1:

**Pull request merged**